faker/providers/misc/en_PH/__init__.py

```diff
@@ -0,0 +1,56 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from .. import Provider as MiscProvider
+
+
+class Provider(MiscProvider):
```

Comment:
I dont see this as a use-case commmon enough to warrant inclusion in `Faker`, but you could package it as a [Community Provider](https://github.com/joke2k/faker/blob/master/docs/communityproviders.rst)

Reply:
Oh, I missed the fact that it was in the address provider. Let's leave it then. 👍